Add python section to sign-into-web-app guide. OKTA-354778 (#1897)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/sign-into-web-app/require-authentication/python/allowanon.md

> Allowing anonymous access for a route just means you do not need to include any of the `@login_required` decorator or some middleware in your route.

Reviewer: ```suggestion
Allowing anonymous access for a route means that you don't need to include the `@login_required` decorator or middleware in your route.
```
